common/test/acceptance/pages/studio/import_export.py

```diff
@@ -171,16 +171,15 @@ def _wait_for_button(self):
             timeout=30
         ).fulfill()
 
-    def upload_tarball(self, tarball_filename):
+    def  upload_tarball(self, tarball_filename):
         """
         Upload a tarball to be imported.
         """
         asset_file_path = self.file_path(tarball_filename)
         # Make the upload elements visible to the WebDriver.
         self.browser.execute_script('$(".file-name-block").show();$(".file-input").show()')
+        self.q(css='input[type="file"]')[0].send_keys(self.file_path('test.tar.gz'))
```

Comment:
can you change the end of the line to `self.file_path(tarball_filename)` instead of the hardcoded `test.tar.gz`? Not sure if that was part of the problem or not but I'm assuming `send_keys` is more resilient & is the crux of your solution.